qubit-sdk/src/main/java/com/qubit/android/sdk/internal/eventtracker/EventTrackerImpl.java

```diff
@@ -331,6 +339,28 @@ public void run() {
       }
       scheduleNextSendEventsTask();
     }
+
+    private void storeEventsToSend(List<EventModel> eventsToSend) {
```

Comment:
I would be happy, if logic specific to placements attributes would be handled somewhere else, not in EventTracker.

Reply:
Moved to new `PlacementAttributesInteractor` class